tests/auth_tests/test_validators.py

```diff
@@ -259,3 +264,73 @@ def test_ascii_validator(self):
             with self.subTest(invalid=invalid):
                 with self.assertRaises(ValidationError):
                     v(invalid)
+
+    def test_username_minimum_length_validator(self):
+        valid_usernames = ['guido', 'raymond']
+        invalid_usernames = ['1t', 'tw', '']
+        v = validators.UsernameMinimumLengthValidator(min_length=3)
+        for valid in valid_usernames:
+            with self.subTest(valid=valid):
+                v(valid)
+        for invalid in invalid_usernames:
+            with self.subTest(invalid=invalid):
+                with self.assertRaises(ValidationError):
```

Comment:
prefer `with self.assertRaisesMessage(ValidationError, msg):` according to this checklist:

https://docs.djangoproject.com/en/3.0/internals/contributing/writing-code/coding-style/